nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -689,6 +689,7 @@ private void configureGenericConnector(Server server, HttpConfiguration configur
                 serverConnector.setHost(hostname);
             }
             serverConnector.setPort(port);
+            serverConnector.setIdleTimeout(60000);
```

Comment:
Assuming these changes are related to the frequency of the automatic browser refresh, should we make this timeout relative to the value configured in `nifi.ui.autorefresh.interval`.